### PR TITLE
C23_WMDE_Mobile_DE_09

### PR DESCRIPTION
**Pull request by Abban**
- Based on CTRL of test 08
- VAR Soft Close has an already donated button

Ticket: https://phabricator.wikimedia.org/T351297

### CODE REVIEW

**Comment by Abban** [comment]
I forgot to add the VAR tests. Will add them in the morning.

File: src/themes/Mikings/SoftClose/SoftClose.scss [hunk 1]
```diff
@@ -46,7 +46,6 @@
 			background-color: colors.$secondary;
 			border-radius: 18px;
 			font-size: 14px;
-			line-height: 40px;
```
Comment (gbirke) [inline src/themes/Mikings/SoftClose/SoftClose.scss:49]: Isn't removing these line-heights a BC-breaking change for the archived banners?
I'm not sure what we should do in cases like these.

**Review by gbirke — APPROVED** [review]
Works and is live.

File: src/themes/Mikings/SoftClose/SoftClose.scss (hunk 1 above)
Comment (Abban) [reply src/themes/Mikings/SoftClose/SoftClose.scss:49]: We had issues that the line height was breaking the text alignment on the buttons, this fixes it. So it'll now also be fixed in the archived banners, which I think is fine. We always have the tags in the repo if we need an exact global state.

**Pull request merged**